source/draw/RasterBgr.ooc

```diff
@@ -117,7 +117,7 @@ RasterBgr: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 3
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
+		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents, true), IntSize2D new(x, y))
```

Comment:
You're right... I think opening image files with StbImage is the only time we can't read into an existing ByteBuffer - it simply has no API for it (which makes sense, since it's also responsible for figuring out how much memory we need to allocate: *It* tells *us* how many channels and pixels are in the image we're trying to open, not the other way around).